### PR TITLE
[Minor] Fix result parsing for SAVAPI

### PR DESCRIPTION
**Pull request by croessner**

### CODE REVIEW

File: src/plugins/lua/antivirus.lua [hunk 1]
```diff
@@ -23,46 +23,6 @@ local redis_params
 
 local N = "antivirus"
 
-if confighelp then
```
Comment (fatalbanana) [inline src/plugins/lua/antivirus.lua]: Why is this removed?

Comment (croessner) [inline src/plugins/lua/antivirus.lua]: That is what I tried to tell you in IRC ;-) You must please cherry-pick :-) The 1.5.7 release does not know about confighelp. That came later.

Comment (fatalbanana) [inline src/plugins/lua/antivirus.lua]: Commits applied to the master branch likely pick apply cleanly to rspamd-1.5. Applying your changes to rspamd-1.5 and then reverse cherry-picking is less likely to work well (but could). Copying files wholesale between branches won't work.

Comment (croessner) [inline src/plugins/lua/antivirus.lua]: Is there something that I can do right now, or can you doe it? If you look at my changes, they are really very little. In fact I kicked the trim() function and changed some lines concerning the 310 parsing. This is much cleaner than before. And hopefully the parsing is fixed now and forever ;-)

Comment (fatalbanana) [inline src/plugins/lua/antivirus.lua]: A well-formed PR would be most helpful. :)
